k8s/Makefile

```diff
@@ -16,11 +16,7 @@ create-helm-charts:
 	kustomize build kustomize/helm-components-ns/ > ${HELM_COMPONENTS_BASE}/seldon-v2-components.yaml
 	kustomize build kustomize/helm-servers/ > ${HELM_SERVERS_BASE}/seldon-v2-servers.yaml
 	sed -i 's/\(.*\)\(imagePullSecrets:\).*/{{- with .Values.imagePullSecrets }}\n\1\2\n\1{{- toYaml . | nindent 8 }}\n{{- end }}/g' ${HELM_COMPONENTS_BASE}/seldon-v2-components.yaml
-	sed -n --file=add-helm-toggle-hodometer.sed ${HELM_COMPONENTS_BASE}/seldon-v2-components.yaml \
```

Comment:
:wrench: As we don't need the magical `sed` script any more, that can be deleted (as much as it saddens me to see it go!)

Reply:
fixed in https://github.com/SeldonIO/seldon-core/pull/5025/commits/a7511fb831865eeedef8c088f4cc99cc034324da